docker-compose.redis.yaml

```
@@ -1,13 +1,19 @@
 #ddev-generated
+volumes:
+  redis:
+    name: ddev-${DDEV_SITENAME}-redis
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+
 services:
   redis:
     container_name: ddev-${DDEV_SITENAME}-redis
-    image: redis:6-bullseye
-    # These labels ensure this service is discoverable by ddev.
+    image: redis:${DDEV_REDIS_VERSION:-7}-alpine
```

Looking into Discord, I thought there was agreement on something like this:

```suggestion
    image: redis:${DDEV_REDIS_VERSION:-6}
```

Or even if we force alpine, it should be like this:

```suggestion
    image: redis:${DDEV_REDIS_VERSION:-7-alpine}
```

---

I missed the swap from 7 to 6. But I'd keep the alpine. No drawbacks, smaller size :)